src/logger.rs

```diff
@@ -0,0 +1,61 @@
+use std::{env, sync::OnceLock};
+
+#[derive(Debug, Clone, Copy)]
+enum FfmpegLogLevel {
+    Error,
+    Warn,
+    Info,
+    Debug,
+}
+
+fn ffmpeg_log_level() -> FfmpegLogLevel {
+    static LOG_LEVEL: OnceLock<FfmpegLogLevel> = OnceLock::new();
+
+    *LOG_LEVEL.get_or_init(|| match env::var("FFMPEG_LOG_LEVEL") {
+        Ok(debug) if debug == "debug" => FfmpegLogLevel::Debug,
+        Ok(info) if info == "info" => FfmpegLogLevel::Info,
+        Ok(warn) if warn == "warn" => FfmpegLogLevel::Warn,
+        Ok(error) if error == "error" => FfmpegLogLevel::Error,
+        Ok(_) => FfmpegLogLevel::Warn,
+        Err(_) => FfmpegLogLevel::Warn,
+    })
```

Comment:
Do we want to use env vars for such configurations?
We have more and more things configurable by env vars in similar way. Isn't it better to use some config file instead?

Reply:
I addressed that in https://github.com/membraneframework/video_compositor/pull/342 description

Reply:
That explanation convinced me 👍